### PR TITLE
feat: add share button

### PR DESCRIPTION
**Pull request by ioane-bendingspoons**
## Description

This PR adds Share functionality

## Checklist

- [x] I have followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md).
- [x] The documentation related to the proposed change has been updated accordingly (plus comments in code).
- [x] I have written new tests for my core changes, as applicable.
- [x] I have successfully run tests with my changes locally.
- [x] It is ready for review! :rocket:

### CODE REVIEW

**Comment by ImmuniDGR** [comment]
🚔 **Safety Check** 🚔

🔰 **Result** 🔰
- The following files have been modified: build.gradle.


⚠️ Some configuration files don't match the master branch. If you did not perform these changes, **please rebase on the master branch**.

🛠 **Diagnostic information** 🛠
- CircleCI scheduler pipeline: #3229 (id: c5ff7264-7ac2-4b15-9465-6eb97d341cdc)
- Last verified commit: ab266d64ce5ae62fb557e261f187cdd73bcaf180
- Time of check: 24/06/2020, 16:02:58 UTC
- The following protected files have been checked for changes: .editorconfig, .gitmodules, Gemfile, Gemfile.lock, app/build.gradle, app/proguard-rules.pro, build.gradle, ci/changelog\_config.tpl, ci/danger/commitlint/index.ts, ci/danger/commitlint/rules.ts, ci/danger/danger.ts, ci/danger/ktlint.ts, dangerfile.ts, fastlane/Appfile, fastlane/Fastfile, gradle.properties, gradlew, gradlew.bat, package.json, publiccode.yml, scheduler\_config.json, settings.gradle, template.properties, yarn.lock.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -306,6 +306,8 @@ The data saved on your smartphone and the connections to the server are encrypte
 Above the line, describe your queries or issues with the app. If possible, do not delete the text below. It will help the Immuni Support team to better assist you. 
 Operating system: iOS 13.5.1; Model: iPhone XS; Exposure notifications: Active; [other useful info, same as the info displayed on the contact support screen]</string>
   <string name="settings_setting_support">Contact Support</string>
+  <string name="settings_setting_share">Condividi l\'app</string>
+  <string name="settings_setting_share_message">Aiutaci a combattere insieme l’epidemia di COVID-19. Scarica anche te Immuni. https://www.immuni.italia.it/</string>
```
Comment (bfabio) [inline app/src/main/res/values/strings.xml:310]: > Scarica anche te Immuni

"Anche tu". "*Anche te*" is a regionalism.

Comment (ioane-bendingspoons) [reply app/src/main/res/values/strings.xml]: Thanks for the input, fixed.

**Comment by ImmuniDGR** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: Commitlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Thank you for submitting a pull request! The team will review your submission as soon as possible.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c1df8afed19fb9aabd1f06a97be1ae646b8f3499
</p>

**Pull request merged**